### DISCUSSION TITLE
How do I confirm my organizer status?

### DISCUSSION THREAD

**Discussion by CarlFK** [pretalx/pretalx#1525] [Q&A] [answered]
I accepted the organizer invite link, which now 404s (presumably because I accepted it) but I can't find something to confirm I have 
organizer privileges.  

my ultimate goal is: 

https://docs.pretalx.org/api/resources/speakers.html
email:  The speaker’s email address. Available if the requesting user has organiser privileges.

There are  a bunch of reasons I'm not getting it, like I'm not doing auth correctly, so no bug report, yet.

**Comment by rixx** [comment]
You will see that you have organiser privileges when you navigate to /orga under your main domain. That's also where you can find the link to your profile page, which includes your auth token.

**Reply by CarlFK** [reply]
https://pretalx.northbaypython.org/orga  redirects me to https://pretalx.northbaypython.org/orga/event/ which shows:
^a^c... paste:
```
The[ pretalx ](https://pretalx.northbaypython.org/orga/)logo pretalx
    Carl Karsten
Events
Your upcoming events
powered by [pretalx](https://pretalx.com/)
```
the event organiser says: 
'I can see that [org priv], I just don't think I can assign you the privilege.`

For this issue, I don't want to dig into how to assign the privilege, just see what my current status is.

It would be nice if I saw something like 
organiser: False
in keeping with
Explicit is better than implicit.

**Reply by rixx** [reply, marked as answer]
Seeing an empty screen like that indicates that you don't have access to any events. Your organiser will need to invite you to a team, and you will have to accept the invitation (the 404 could also be an indication that the invite was removed, or that you accepted it with a different account).

I added a new dashboard tile in 06cc84e6.
